conf/base.config

```diff
@@ -22,20 +22,17 @@ process {
   withName: fastqc {
     errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'ignore' }
```

Comment:
What is this ignoring of error messages for?

Reply:
clashing of sample names when run with --multipleSequencingRuns

Reply:
and ignoring is the right strategy?

Reply:
As discussed, its not. I'll address this soon but not in this release.

Reply:
Issue opened: [#180](https://github.com/nf-core/ampliseq/issues/180)